packages/common/src/types/catalog.ts

```diff
@@ -53,6 +53,8 @@ type CustomIcon = {
 
 export type CatalogItemIcon = EmojiIcon | CustomIcon;
 
+export const UNCATEGORIZED_TAG_UUID = '__uncategorized__';
```

Comment:
do you feel like this is problematic? I dont think so, since we use the taguuids